### PR TITLE
[Release] Cascade RCNN res101 document model support

### PR DESCRIPTION
**Pull request by yekruglov**
Ticket: CVS-51961

[PR](https://github.com/openvinotoolkit/openvino/pull/10902) to master branch

### CODE REVIEW

File: docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_Cascade_RCNN_res101.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+* Download the pre-trained [model](https://download.openmmlab.com/mmdetection/v2.0/cascade_rcnn/cascade_rcnn_r101_fpn_1x_coco/cascade_rcnn_r101_fpn_1x_coco_20200317-0b6a2fbf.pth). Link to the model can also be found [here](https://github.com/open-mmlab/mmdetection/blob/master/configs/cascade_rcnn/README.md).
+
+* To convert model to ONNX* format use this [script](https://github.com/open-mmlab/mmdetection/blob/master/tools/deployment/pytorch2onnx.py).
+
+```bash
+python3 tools/deployment/pytorch2onnx.py configs/cascade_rcnn/cascade_rcnn_r101_fpn_1x_coco.py cascade_rcnn_r101_fpn_1x_coco_20200317-0b6a2fbf.pth --output-file cascade_rcnn_r101_fpn_1x_coco.onnx
+```
+
+The script generates ONNX* model file `cascade_rcnn_r101_fpn_1x_coco.onnx`.
```
Comment (achetver) [inline docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_Cascade_RCNN_res101.md:22]: Do we need to specify where this file can be found by user?

File: docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_Cascade_RCNN_res101.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+## Download and convert model to ONNX*
+
+* Clone the [repository](https://github.com/open-mmlab/mmdetection):
+
+```bash
+git clone https://github.com/open-mmlab/mmdetection
+cd mmdetection
+```
+
+> **NOTE**: To set up an environment refer to this [instruction](https://github.com/open-mmlab/mmdetection/blob/master/docs/en/get_started.md#installation).
```
Comment (tsavina) [inline docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_Cascade_RCNN_res101.md:12]:
```suggestion
> **NOTE**: To set up an environment, refer to this [instruction](https://github.com/open-mmlab/mmdetection/blob/master/docs/en/get_started.md#installation).
```

File: docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_Cascade_RCNN_res101.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Convert PyTorch* Cascade RCNN R-101 Model {#openvino_docs_MO_DG_prepare_model_convert_model_pytorch_specific_Convert_Cascade_RCNN_res101}
+
+## Download and convert model to ONNX*
```
Comment (tsavina) [inline docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_Cascade_RCNN_res101.md:3]:
```suggestion
## Download and Convert Model to ONNX
```

File: docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_Cascade_RCNN_res101.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+```bash
+git clone https://github.com/open-mmlab/mmdetection
+cd mmdetection
+```
+
+> **NOTE**: To set up an environment refer to this [instruction](https://github.com/open-mmlab/mmdetection/blob/master/docs/en/get_started.md#installation).
+
+* Download the pre-trained [model](https://download.openmmlab.com/mmdetection/v2.0/cascade_rcnn/cascade_rcnn_r101_fpn_1x_coco/cascade_rcnn_r101_fpn_1x_coco_20200317-0b6a2fbf.pth). Link to the model can also be found [here](https://github.com/open-mmlab/mmdetection/blob/master/configs/cascade_rcnn/README.md).
+
+* To convert model to ONNX* format use this [script](https://github.com/open-mmlab/mmdetection/blob/master/tools/deployment/pytorch2onnx.py).
```
Comment (tsavina) [inline docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_Cascade_RCNN_res101.md:16]:
```suggestion
* To convert the model to ONNX format, use this [script](https://github.com/open-mmlab/mmdetection/blob/master/tools/deployment/pytorch2onnx.py).
```

File: docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_Cascade_RCNN_res101.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# Convert PyTorch* Cascade RCNN R-101 Model {#openvino_docs_MO_DG_prepare_model_convert_model_pytorch_specific_Convert_Cascade_RCNN_res101}
```
Comment (tsavina) [inline docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_Cascade_RCNN_res101.md:1]:
```suggestion
# Convert PyTorch Cascade RCNN R-101 Model {#openvino_docs_MO_DG_prepare_model_convert_model_pytorch_specific_Convert_Cascade_RCNN_res101}
```

File: docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_Cascade_RCNN_res101.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+* Clone the [repository](https://github.com/open-mmlab/mmdetection):
+
+```bash
+git clone https://github.com/open-mmlab/mmdetection
+cd mmdetection
+```
+
+> **NOTE**: To set up an environment refer to this [instruction](https://github.com/open-mmlab/mmdetection/blob/master/docs/en/get_started.md#installation).
+
+* Download the pre-trained [model](https://download.openmmlab.com/mmdetection/v2.0/cascade_rcnn/cascade_rcnn_r101_fpn_1x_coco/cascade_rcnn_r101_fpn_1x_coco_20200317-0b6a2fbf.pth). Link to the model can also be found [here](https://github.com/open-mmlab/mmdetection/blob/master/configs/cascade_rcnn/README.md).
```
Comment (tsavina) [inline docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_Cascade_RCNN_res101.md:14]:
```suggestion
* Download the pre-trained [model](https://download.openmmlab.com/mmdetection/v2.0/cascade_rcnn/cascade_rcnn_r101_fpn_1x_coco/cascade_rcnn_r101_fpn_1x_coco_20200317-0b6a2fbf.pth). You can also find the link to the model [here](https://github.com/open-mmlab/mmdetection/blob/master/configs/cascade_rcnn/README.md).
```

File: docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_Cascade_RCNN_res101.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+* To convert model to ONNX* format use this [script](https://github.com/open-mmlab/mmdetection/blob/master/tools/deployment/pytorch2onnx.py).
+
+```bash
+python3 tools/deployment/pytorch2onnx.py configs/cascade_rcnn/cascade_rcnn_r101_fpn_1x_coco.py cascade_rcnn_r101_fpn_1x_coco_20200317-0b6a2fbf.pth --output-file cascade_rcnn_r101_fpn_1x_coco.onnx
+```
+
+The script generates ONNX* model file `cascade_rcnn_r101_fpn_1x_coco.onnx` in the directory `tools/deployment/`. If you need, you can specify model name or output directory using `--output-file <path-to-dir>/<model-name>.onnx`
```
Comment (tsavina) [inline docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_Cascade_RCNN_res101.md:22]:
```suggestion
The script generates ONNX model file `cascade_rcnn_r101_fpn_1x_coco.onnx` in the directory `tools/deployment/`. If required, you can specify the model name or output directory using `--output-file <path-to-dir>/<model-name>.onnx` 
```

File: docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_Cascade_RCNN_res101.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+* To convert model to ONNX* format use this [script](https://github.com/open-mmlab/mmdetection/blob/master/tools/deployment/pytorch2onnx.py).
+
+```bash
+python3 tools/deployment/pytorch2onnx.py configs/cascade_rcnn/cascade_rcnn_r101_fpn_1x_coco.py cascade_rcnn_r101_fpn_1x_coco_20200317-0b6a2fbf.pth --output-file cascade_rcnn_r101_fpn_1x_coco.onnx
+```
+
+The script generates ONNX* model file `cascade_rcnn_r101_fpn_1x_coco.onnx` in the directory `tools/deployment/`. If you need, you can specify model name or output directory using `--output-file <path-to-dir>/<model-name>.onnx` 
+
+## Convert ONNX* Cascade RCNN R-101 Model to IR
```
Comment (tsavina) [inline docs/MO_DG/prepare_model/convert_model/pytorch_specific/Convert_Cascade_RCNN_res101.md:24]:
```suggestion
## Convert ONNX Cascade RCNN R-101 Model to IR
```

**Review by tsavina — APPROVED** [review]

**Pull request merged**